### ISSUE TITLE
Crash report on GrapheneOS/Pixel 4a

### ISSUE THREAD

**Issue by blogdemoi** [Catfriend1/syncthing-android#923]
### Description of the issue
Crash when opening the app
### Reproduction Steps
Open the app
### Version Information
- App Version: 1.21.0.2
- Android Version: GrapheneOS latest stable
- Device manufacturer: Google
- Device model: Pixel 4a

### Android Log
```
type: crash
osVersion: google/sunfish/sunfish:13/TP1A.221005.002/2022100300:user/release-keys
package: com.github.catfriend1.syncthingandroid:1210002
process: com.github.catfriend1.syncthingandroid

java.lang.ArrayIndexOutOfBoundsException: length=1; index=2
at com.nutomic.syncthingandroid.util.Util.readableFileSize(Util.java:72)
at com.nutomic.syncthingandroid.views.FoldersAdapter.updateFolderStatusView(FoldersAdapter.java:214)
at com.nutomic.syncthingandroid.views.FoldersAdapter.getView(FoldersAdapter.java:84)
at android.widget.AbsListView.obtainView(AbsListView.java:2398)
at android.widget.ListView.makeAndAddView(ListView.java:2067)
at android.widget.ListView.fillSpecific(ListView.java:1492)
at android.widget.ListView.layoutChildren(ListView.java:1802)
at android.widget.AbsListView.onLayout(AbsListView.java:2195)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at androidx.viewpager.widget.ViewPager.onLayout(ViewPager.java:1775)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at androidx.drawerlayout.widget.DrawerLayout.onLayout(DrawerLayout.java:1231)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at com.android.internal.policy.DecorView.onLayout(DecorView.java:797)
at android.view.View.layout(View.java:23694)
at android.view.ViewGroup.layout(ViewGroup.java:6413)
at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3911)
at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3298)
at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2286)
at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8948)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1231)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
at android.view.Choreographer.doCallbacks(Choreographer.java:899)
at android.view.Choreographer.doFrame(Choreographer.java:832)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
at android.os.Handler.handleCallback(Handler.java:942)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:201)
at android.os.Looper.loop(Looper.java:288)
at android.app.ActivityThread.main(ActivityThread.java:7904)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
at com.android.internal.os.ExecInit.main(ExecInit.java:49)
at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)

```

**Comment by fhunter** [comment]
This looks like same crash as https://github.com/Catfriend1/syncthing-android/issues/921

**Comment by Catfriend1** [comment]
Is this solved by updating to https://github.com/Catfriend1/syncthing-android/releases/tag/v1.21.0.3 ?

**Comment by KevinMX** [comment]
> Is this solved by updating to https://github.com/Catfriend1/syncthing-android/releases/tag/v1.21.0.3 ?

Redmi K50 / rubens / MIUI 13.0.24 / Android 12 user here, crashing on 1.21.0.2 (from Google Play), can confirm v1.21.0.3 solves the issue.

**Comment by Catfriend1** [comment]
:-)

**Comment by Iey4iej3** [comment]
Also solved for me in 1.21.0.3 (LOS 19.1)
